### ISSUE TITLE
USB issue Mac OS

### ISSUE THREAD

**Issue by fabulouspanda** [luke-jr/bfgminer#311]
Using version 3.5.0 on Mac OS I get the following errors:

```
     [2013-11-06 12:53:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
     [2013-11-06 12:53:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
```

...
         [2013-11-06 12:53:57] usb_devinfo_scan: Error getting USB string 3 (iSerialNumber): LIBUSB_ERROR_OTHER

This is with a block erupter usb and a jalapeño, little single SC and single SC connected and the drivers installed, which worked in previous versions

**Comment by fabulouspanda** [comment]
A little better on 3.5.1, the block erupter works but the BFL devices claim to already be in use…

```
   [2013-11-09 08:55:57] /dev/cu.usbserial-FTWP5BMQ is already in use by another process
    [2013-11-09 08:55:57] ModMiner detect: failed to open /dev/cu.usbserial-FTWP5BMQ
    [2013-11-09 08:55:57] /dev/cu.usbserial-FTWWXWE8 is already in use by another process
    [2013-11-09 08:55:57] ModMiner detect: failed to open /dev/cu.usbserial-FTWWXWE8
    [2013-11-09 08:55:57] /dev/cu.usbserial-FTWWH387 is already in use by another process
    [2013-11-09 08:55:57] ModMiner detect: failed to open /dev/cu.usbserial-FTWWH387
```

I made sure they weren't, unplugged and replugged etc. - is it because it thinks it sees modminer? it repeats the already in use message for each device again, only missing the modminer line.

**Comment by luke-jr** [comment]
Full output from:

```
bfgminer -D -d?
```

please

**Comment by fabulouspanda** [comment]
Here we go:

```
 [2013-11-09 09:09:56] Timers: Using gettimeofday
 [2013-11-09 09:09:56] Global quota greatest common denominator set to 1
 [2013-11-09 09:09:56] Global quota greatest common denominator set to 1
 [2013-11-09 09:09:56] Global quota greatest common denominator set to 1
 [2013-11-09 09:09:56] setrlimit: Changed soft fd limit from 2560 to 1024 (FD_SETSIZE=1024; hard limit=unlimited)
 [2013-11-09 09:09:56] Started bfgminer 3.5.1
 [2013-11-09 09:09:56] Loaded configuration file /Users/admin/Library/Application Support/MacMiner/bfgurls.conf
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 3 (iSerialNumber): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 1 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 2 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] usb_devinfo_scan: Error getting USB string 3 (iSerialNumber): LIBUSB_ERROR_OTHER
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:003 (path=(null), vid=0403, pid=6014, manuf=Butterfly Labs, prod=BitFORCE SHA256 SC, serial=FTWP5BMQ)
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:011 (path=(null), vid=04f9, pid=01f1, manuf=Brother, prod=DCP-6690CW, serial=BROA0F313146)
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:058:002 (path=(null), vid=10c4, pid=ea60, manuf=Silicon Labs, prod=CP2102 USB to UART Bridge Controller, serial=0001)
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:010 (path=(null), vid=0403, pid=6014, manuf=Butterfly Labs, prod=BitFORCE SHA256 SC, serial=FTWWXWE8)
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:250:002 (path=(null), vid=0403, pid=6014, manuf=Butterfly Labs, prod=BitFORCE SHA256 SC, serial=FTWWH387)
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:009 (path=(null), vid=05ac, pid=0304, manuf=Mitsumi Electric, prod=Apple Optical USB Mouse, serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:008 (path=(null), vid=05ac, pid=0250, manuf=Apple Inc., prod=Apple Keyboard, serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:006 (path=(null), vid=05ac, pid=9226, manuf=Apple Inc., prod=Apple LED Cinema Display, serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:005 (path=(null), vid=05ac, pid=8508, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:004 (path=(null), vid=05ac, pid=1105, manuf=Apple Inc., prod=Display Audio, serial=AE001116)
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:007 (path=(null), vid=05ac, pid=1006, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:002 (path=(null), vid=05ac, pid=9126, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:093:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:061:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:029:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:090:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:026:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:058:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:253:001 (path=(null), vid=05ac, pid=8006, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found usb device at usb:250:001 (path=(null), vid=05ac, pid=8006, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:2d000012 (path=/dev/cu.usbserial-FTWP5BMQ, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:2b000012 (path=/dev/cu.SLAB_USBtoUART, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:23000012 (path=/dev/cu.usbserial-FTWWXWE8, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:1f000012 (path=/dev/cu.usbserial-FTWWH387, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:0b000012 (path=/dev/cu.Bluetooth-PDA-Sync, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:09000012 (path=/dev/cu.Bluetooth-Modem, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:07000012 (path=/dev/cu.Bluetooth-Serial-2, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] lowlevel_scan: Found vcom device at dev_t:05000012 (path=/dev/cu.Bluetooth-Serial-1, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:09:57] DISPLAY not set, setting :0 just in case
 [2013-11-09 09:09:57] CL Platform 0 vendor: Apple
 [2013-11-09 09:09:57] CL Platform 0 name: Apple
 [2013-11-09 09:09:57] CL Platform 0 version: OpenCL 1.2 (Apr 25 2013 18:32:06)
 [2013-11-09 09:09:57] Platform 0 devices: 1
 [2013-11-09 09:09:57]  0   ATI Radeon HD 5770
 [2013-11-09 09:09:57] Unable to load ati adl library
 [2013-11-09 09:09:57] Devices detected:
 [2013-11-09 09:09:57]   0. OCL 0  (driver: opencl)
1 devices listed
 [2013-11-09 09:09:57] Devices detected:                    
 [2013-11-09 09:09:57]   0. OCL 0  (driver: opencl)                    
```

**Comment by luke-jr** [comment]
Add any -S options you might be using to it, and paste the same.

**Comment by fabulouspanda** [comment]
Ah sorry, here it is with -S all as well

```
 [2013-11-09 09:21:40] Timers: Using gettimeofday
 [2013-11-09 09:21:40] Global quota greatest common denominator set to 1
 [2013-11-09 09:21:40] Global quota greatest common denominator set to 1
 [2013-11-09 09:21:40] Global quota greatest common denominator set to 1
 [2013-11-09 09:21:40] setrlimit: Changed soft fd limit from 2560 to 1024 (FD_SETSIZE=1024; hard limit=unlimited)
 [2013-11-09 09:21:40] Started bfgminer 3.5.1
 [2013-11-09 09:21:40] Loaded configuration file /Users/admin/Library/Application Support/MacMiner/bfgurls.conf
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 3 (iSerialNumber): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 1 (iManufacturer): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 2 (iProduct): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] usb_devinfo_scan: Error getting USB string 3 (iSerialNumber): LIBUSB_ERROR_OTHER
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:003 (path=(null), vid=0403, pid=6014, manuf=Butterfly Labs, prod=BitFORCE SHA256 SC, serial=FTWP5BMQ)
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:011 (path=(null), vid=04f9, pid=01f1, manuf=Brother, prod=DCP-6690CW, serial=BROA0F313146)
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:058:002 (path=(null), vid=10c4, pid=ea60, manuf=Silicon Labs, prod=CP2102 USB to UART Bridge Controller, serial=0001)
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:010 (path=(null), vid=0403, pid=6014, manuf=Butterfly Labs, prod=BitFORCE SHA256 SC, serial=FTWWXWE8)
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:250:002 (path=(null), vid=0403, pid=6014, manuf=Butterfly Labs, prod=BitFORCE SHA256 SC, serial=FTWWH387)
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:009 (path=(null), vid=05ac, pid=0304, manuf=Mitsumi Electric, prod=Apple Optical USB Mouse, serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:008 (path=(null), vid=05ac, pid=0250, manuf=Apple Inc., prod=Apple Keyboard, serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:006 (path=(null), vid=05ac, pid=9226, manuf=Apple Inc., prod=Apple LED Cinema Display, serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:005 (path=(null), vid=05ac, pid=8508, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:004 (path=(null), vid=05ac, pid=1105, manuf=Apple Inc., prod=Display Audio, serial=AE001116)
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:007 (path=(null), vid=05ac, pid=1006, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:002 (path=(null), vid=05ac, pid=9126, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:093:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:061:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:029:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:090:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:026:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:058:001 (path=(null), vid=05ac, pid=8005, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:253:001 (path=(null), vid=05ac, pid=8006, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found usb device at usb:250:001 (path=(null), vid=05ac, pid=8006, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:2d000012 (path=/dev/cu.usbserial-FTWP5BMQ, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:2b000012 (path=/dev/cu.SLAB_USBtoUART, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:23000012 (path=/dev/cu.usbserial-FTWWXWE8, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:1f000012 (path=/dev/cu.usbserial-FTWWH387, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:0b000012 (path=/dev/cu.Bluetooth-PDA-Sync, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:09000012 (path=/dev/cu.Bluetooth-Modem, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:07000012 (path=/dev/cu.Bluetooth-Serial-2, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] lowlevel_scan: Found vcom device at dev_t:05000012 (path=/dev/cu.Bluetooth-Serial-1, vid=0000, pid=0000, manuf=(null), prod=(null), serial=(null))
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:41] Icarus Read: No data in 0.10 seconds
 [2013-11-09 09:21:41] Icarus Detect: Test failed at /dev/cu.usbserial-FTWP5BMQ: get 00000000, should: 000187a2
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.SLAB_USBtoUART
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.SLAB_USBtoUART
 [2013-11-09 09:21:41] Icarus Detect: Test succeeded at /dev/cu.SLAB_USBtoUART: got 000187a2
 [2013-11-09 09:21:41] Found ICA 0 at /dev/cu.SLAB_USBtoUART
 [2013-11-09 09:21:41] ICA 0: Init: baud=115200 work_division=0 fpga_count=0
 [2013-11-09 09:21:41] ICA 0: Init: mode=default read_count=50 limit=0ms Hs=2.640830e-09
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:41] Icarus Read: No data in 0.10 seconds
 [2013-11-09 09:21:41] Icarus Detect: Test failed at /dev/cu.usbserial-FTWWXWE8: get 00000000, should: 000187a2
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:41] Icarus Read: No data in 0.10 seconds
 [2013-11-09 09:21:41] Icarus Detect: Test failed at /dev/cu.usbserial-FTWWH387: get 00000000, should: 000187a2
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:41] Icarus Read: No data in 0.10 seconds
 [2013-11-09 09:21:41] Icarus Detect: Test failed at /dev/cu.Bluetooth-PDA-Sync: get 00000000, should: 000187a2
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.Bluetooth-Modem
 [2013-11-09 09:21:41] Open /dev/cu.Bluetooth-Modem failed: Resource busy
 [2013-11-09 09:21:41] Icarus Detect: Failed to open /dev/cu.Bluetooth-Modem
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:41] Icarus Read: No data in 0.10 seconds
 [2013-11-09 09:21:41] Icarus Detect: Test failed at /dev/cu.Bluetooth-Serial-2: get 00000000, should: 000187a2
 [2013-11-09 09:21:41] Icarus Detect: Attempting to open /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:41] Icarus Read: No data in 0.10 seconds
 [2013-11-09 09:21:41] Icarus Detect: Test failed at /dev/cu.Bluetooth-Serial-1: get 00000000, should: 000187a2
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:41] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:43] Acquired exclusive advisory lock on /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:43] Open /dev/cu.Bluetooth-Modem failed: Resource busy
 [2013-11-09 09:21:43] Acquired exclusive advisory lock on /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:43] Acquired exclusive advisory lock on /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:43] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:43] littlefury: Successfully opened /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:44] Read of fd 8 returned 0
 [2013-11-09 09:21:44] littlefury: DEVPROTO: RECV 
 [2013-11-09 09:21:44] littlefury: Failed to read correct packet header
 [2013-11-09 09:21:44] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:44] littlefury: Successfully opened /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:45] Read of fd 9 returned 0
 [2013-11-09 09:21:45] littlefury: DEVPROTO: RECV 
 [2013-11-09 09:21:45] littlefury: Failed to read correct packet header
 [2013-11-09 09:21:45] Acquired exclusive advisory lock on /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:45] littlefury: Successfully opened /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:46] Read of fd 10 returned 0
 [2013-11-09 09:21:46] littlefury: DEVPROTO: RECV 
 [2013-11-09 09:21:46] littlefury: Failed to read correct packet header
 [2013-11-09 09:21:46] Acquired exclusive advisory lock on /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:46] littlefury: Successfully opened /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:47] Read of fd 11 returned 0
 [2013-11-09 09:21:47] littlefury: DEVPROTO: RECV 
 [2013-11-09 09:21:47] littlefury: Failed to read correct packet header
 [2013-11-09 09:21:47] Open /dev/cu.Bluetooth-Modem failed: Resource busy
 [2013-11-09 09:21:47] littlefury: Failed to open /dev/cu.Bluetooth-Modem
 [2013-11-09 09:21:47] Acquired exclusive advisory lock on /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:47] littlefury: Successfully opened /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:48] Read of fd 12 returned 0
 [2013-11-09 09:21:48] littlefury: DEVPROTO: RECV 
 [2013-11-09 09:21:48] littlefury: Failed to read correct packet header
 [2013-11-09 09:21:48] Acquired exclusive advisory lock on /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:48] littlefury: Successfully opened /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:49] Read of fd 13 returned 0
 [2013-11-09 09:21:49] littlefury: DEVPROTO: RECV 
 [2013-11-09 09:21:49] littlefury: Failed to read correct packet header
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWP5BMQ is already in use by another process
 [2013-11-09 09:21:49] ModMiner detect: failed to open /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWXWE8 is already in use by another process
 [2013-11-09 09:21:49] ModMiner detect: failed to open /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWH387 is already in use by another process
 [2013-11-09 09:21:49] ModMiner detect: failed to open /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-PDA-Sync is already in use by another process
 [2013-11-09 09:21:49] ModMiner detect: failed to open /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:49] Open /dev/cu.Bluetooth-Modem failed: Resource busy
 [2013-11-09 09:21:49] ModMiner detect: failed to open /dev/cu.Bluetooth-Modem
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-2 is already in use by another process
 [2013-11-09 09:21:49] ModMiner detect: failed to open /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-1 is already in use by another process
 [2013-11-09 09:21:49] ModMiner detect: failed to open /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWP5BMQ is already in use by another process
 [2013-11-09 09:21:49] BFL: Attempting to open /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:49] BFL: Failed to open /dev/cu.usbserial-FTWP5BMQ
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWXWE8 is already in use by another process
 [2013-11-09 09:21:49] BFL: Attempting to open /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:49] BFL: Failed to open /dev/cu.usbserial-FTWWXWE8
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWH387 is already in use by another process
 [2013-11-09 09:21:49] BFL: Attempting to open /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:49] BFL: Failed to open /dev/cu.usbserial-FTWWH387
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-PDA-Sync is already in use by another process
 [2013-11-09 09:21:49] BFL: Attempting to open /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:49] BFL: Failed to open /dev/cu.Bluetooth-PDA-Sync
 [2013-11-09 09:21:49] Open /dev/cu.Bluetooth-Modem failed: Resource busy
 [2013-11-09 09:21:49] BFL: Attempting to open /dev/cu.Bluetooth-Modem
 [2013-11-09 09:21:49] BFL: Failed to open /dev/cu.Bluetooth-Modem
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-2 is already in use by another process
 [2013-11-09 09:21:49] BFL: Attempting to open /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:49] BFL: Failed to open /dev/cu.Bluetooth-Serial-2
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-1 is already in use by another process
 [2013-11-09 09:21:49] BFL: Attempting to open /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:49] BFL: Failed to open /dev/cu.Bluetooth-Serial-1
 [2013-11-09 09:21:49] Devices detected:
 [2013-11-09 09:21:49]   0. ICA 0  (driver: icarus)
1 devices listed
 [2013-11-09 09:21:44] Read of fd 8 returned 0                    
 [2013-11-09 09:21:45] Read of fd 9 returned 0                    
 [2013-11-09 09:21:46] Read of fd 10 returned 0                    
 [2013-11-09 09:21:47] Read of fd 11 returned 0                    
 [2013-11-09 09:21:48] Read of fd 12 returned 0                    
 [2013-11-09 09:21:49] Read of fd 13 returned 0                    
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWP5BMQ is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWXWE8 is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWH387 is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-PDA-Sync is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-2 is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-1 is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWP5BMQ is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWXWE8 is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.usbserial-FTWWH387 is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-PDA-Sync is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-2 is already in use by another process                    
 [2013-11-09 09:21:49] /dev/cu.Bluetooth-Serial-1 is already in use by another process                    
 [2013-11-09 09:21:49] Devices detected:                    
 [2013-11-09 09:21:49]   0. ICA 0  (driver: icarus)                    
```

**Comment by luke-jr** [comment]
Should be fixed in 1a98231ba438fb8c6128eaf832d13cb778f1c676; please confirm and close

**Comment by fabulouspanda** [comment]
Sorry for the late response - This issue is fixed in 3.6 - thanks Luke!
